src/state/session/agent.ts

```diff
@@ -52,8 +52,18 @@ export async function createAgentAndResume(
   } else {
     agent.session = prevSession
     if (!storedAccount.deactivated) {
-      // Intentionally not awaited to unblock the UI:
-      networkRetry(3, () => agent.resumeSession(prevSession))
+      try {
+        // Intentionally not awaited to unblock the UI:
+        networkRetry(3, () => agent.resumeSession(prevSession))
+      } catch (e: any) {
```

Comment:
This isn't awaited so I think you want to do a `.catch()` instead — try/catch won't catch it

Reply:
🤦